.github/workflows/ci.yml

```diff
@@ -157,6 +157,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        # test recent mypy versions on 3.11 and mypy 1.0.1 on all supported python versions
```

Comment:
it says `python-version: ['3.10']` right below, should this be 3.11 now? Or should this comment say 3.10? Not sure if I'm misunderstanding what's happening.